teamwork/templates/base.html

```diff
@@ -1,23 +1,78 @@
-    <link rel="stylesheet" href="{% static 'css/base.css' %}">
-  </head>
-  <body>
-    {% block content %}
-    {% endblock %}
-  </body>
-</html>
+      <!-- Custom CSS -->
+      <link rel="stylesheet" href="{% static 'css/main.css' %}">
+    </head>
+    <body>
+      <!-- Navigation bar -->
+      <nav class="navbar navbar-inverse navbar-static-top" role="navigation">
+                      <div class="container">
+                          <div class="navbar-header">
+                              <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+                                  <span class="sr-only">Toggle navigation</span>
+                                  <span class="icon-bar"></span>
+                                  <span class="icon-bar"></span>
+                                  <span class="icon-bar"></span>
+                              </button>
+                              <p class="navbar-brand">{{ project.name }}</p>
+                          </div>
+                          <div class="navbar-collapse collapse">
+                              <ul class="nav navbar-nav">
+                                  <li class="active"><a href="{% url 'dashboard' user.username %}">Dashboard</a>
+                                  </li>
+                                  <li><a href="calendar">Calendar</a>
+                                  </li>
+                                  <li><button type="button" class="btn btn btn-default btn-success navbar-btn"> <span class="glyphicon glyphicon-plus"></span> Add new project </button>
+                                  </li>
+                              </ul>
+
+                              <ul class="nav navbar-nav navbar-right">
+                                <li class="dropdown">
+                                    <a href="#" class="dropdown-toggle" data-toggle="dropdown"><span class="glyphicon glyphicon-user"></span> {{ user.username }} <b class="caret"></b></a>
+                                    <ul class="dropdown-menu">
+                                        <li><a href="#"><span class="glyphicon glyphicon-cog"></span> Settings</a>
+                                        </li>
+                                        <li class="divider"></li>
+                                        <li><a href="#">Log out</a>
+                                        </li>
+                                    </ul>
+                                </li>
+                              </ul>
+                          </div>
+                          <!--/.nav-collapse -->
+                      </div>
+        </nav>
+      {% block content %}
+      {% endblock %}
+
+      <!--Footer -->
+      <footer class="container-fluid text-center">
+        <p><a href="https://mycourses.aalto.fi/" title="MyCourse" target="_blank">My Course</a>  |  All rights reserved</p>

```

Comment:
I'm also a bit hesitant about the "All rights reserved". Can this be interpreted that we have all right on MyCourses because it's next to it? And copyright laws apply by default so we don't really need to specifically say that we have the rights (I think).